common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/rest/client/vrli/RestClientVrliV2.java

```diff
@@ -38,29 +37,31 @@
 import java.util.stream.Collectors;
 
 public class RestClientVrliV2 extends AbstractRestClientVrli {
```

Comment:
I think it's prudent for V2 to extend V1 and do overwrites where needed?

Reply:
The idea is that V2 will be totally different in term of models. From my perspective doesn't make sense.

Reply:
I agree with @konstantin-nachev